json_blacklist

```diff
@@ -86,38 +85,26 @@ data/json/mapgen/basement/basement_game.json
 data/json/mapgen/basement/basement_lab_stairs.json
 data/json/mapgen/basement/basement_survival.json
 data/json/mapgen/bowling_alley.json
-data/json/mapgen/cabin00.json
-data/json/mapgen/cabin_strange.json
+data/json/mapgen/cabin.json
```

Comment:
This line can probably safely be removed since this file has now been linted.
Same for lines 91, 95, 98, 105, 107, 129, 131, 153, 165, 174, 175, 176, 177, 178, 179, and 180.